### DISCUSSION TITLE
1.0.0.beta版本安装后,如果重启服务器laf-server会启不来

### DISCUSSION THREAD

**Discussion by nightwhite** [labring/laf#688] [新春贺岁版反馈专区]
1.0.0.beta版本安装后,如果重启服务器laf-server会启不来,仅测试本地虚拟机

**Comment by LeezQ** [comment]
有更多的日志信息吗？

**Reply by nightwhite** [reply]
![image](https://user-images.githubusercontent.com/22722256/215658287-565c04eb-3c0f-4c82-b054-fc712af16e86.png)

**Comment by maslow** [comment]
是 mongo 配置的问题，已在新版本中修复。
